src/subcommand/wallet/inscribe.rs

```diff
@@ -89,8 +89,8 @@ impl Inscribe {
     let parent_info;
     let sat;
 
-    match (self.file, self.batch) {
-      (Some(file), None) => {
+    match (self.file, self.batch, self.delegate) {
+      (Some(file), None, _) => {
```

Comment:
This would mean if we have a file and a delegate it would still inscribe the file. Same with the batch below. This should be tested.